composer.json

```diff
@@ -1,16 +1,17 @@
 {
     "name": "mygento/module-kkm",
     "type": "magento2-module",
-    "version": "2.0.",
+    "version": "2.3.0",
     "license": "OSL-3.0",
     "description": "Модуль интеграции фискальных кассовых аппаратов для Magento в соответствии с 54-ФЗ",
     "keywords": [
         "magento",
         "atol"
     ],
     "require": {
-        "php": "^7.0.0",
-        "mygento/base": "^2.2.14"
+        "php": "^7.1.3",
```

Comment:
base has it

Reply:
base doesn't has it

Reply:
https://github.com/magento/magento2/blob/2.3-develop/lib/internal/Magento/Framework/composer.json#L13